openedx/core/djangoapps/credit/models.py

```diff
@@ -8,20 +8,53 @@
 
 import logging
 
+from django.db.utils import IntegrityError
 from django.db import models
+from jsonfield.fields import JSONField
+
 from model_utils.models import TimeStampedModel
+from openedx.core.djangoapps.credit.exceptions import InvalidCreditRequirements
 from xmodule_django.models import CourseKeyField
-from jsonfield.fields import JSONField
+
 
 log = logging.getLogger(__name__)
 
 
 class CreditCourse(models.Model):
-    """Model for tracking a credit course."""
+    """Model for tracking the credit course."""
 
     course_key = CourseKeyField(max_length=255, db_index=True, unique=True)
     enabled = models.BooleanField(default=False)
 
+    @classmethod
+    def is_credit_course(cls, course_key):
+        """ Check that given course is credit or not
+
+        Args:
+            cls(CreditCourse): The class name
```

Comment:
This isn't necessary, just as listing `self` as an argument to a method isn't necessary. `cls` is a conventional argument for a `classmethod`.